neo/public/stylesheets/main.css

```diff
@@ -31,11 +34,10 @@ body {
 }
 
 time-series {
-  position: absolute;
-  top: 82%;
-  left: 15%;
-  width: 80%;
-  height: 15%;
+  z-index: 1000;
+  pointer-events: none;
```

Comment:
It's a way to specify the behavior when the mouse hovers over the DOM. In this case, my purpose was to avoid the count div from blocking the mouse dragging the map. But I just found a bug, I will fix it.